src/metrics-parse.go

```diff
@@ -1,20 +1,36 @@
 package main
 
 import (
+	"errors"
 	"fmt"
+	"regexp"
 	"strconv"
 
+	"github.com/blang/semver/v4"
 	"github.com/newrelic/infra-integrations-sdk/v3/data/inventory"
 	"github.com/newrelic/infra-integrations-sdk/v3/data/metric"
 	"github.com/newrelic/infra-integrations-sdk/v3/log"
 )
 
 const (
-	inventoryQuery = "SHOW GLOBAL VARIABLES"
-	metricsQuery   = "SHOW /*!50002 GLOBAL */ STATUS"
-	replicaQuery   = "SHOW SLAVE STATUS"
+	inventoryQuery  = "SHOW GLOBAL VARIABLES"
+	metricsQuery    = "SHOW /*!50002 GLOBAL */ STATUS"
+	replicaQuery560 = "SHOW SLAVE STATUS"
+	replicaQuery800 = "SHOW REPLICA STATUS"
```

Comment:
> Is this only for version 5.6.0? I'd assume this will be same for 5.7.x also. If so, can we update the name to be a bit more descriptive. Right now it seems that this will only work for 5.6.0.

Similar point here.

Reply:
this should be `replicaQueryForVersion8AndAbove`

Reply:
Resolved it in this [commit](https://github.com/newrelic/nri-mysql/pull/170/commits/fe29f92e5ad2d923fd031d278d2ecb482b0e69be)